conf/defaults.ini

```diff
@@ -576,6 +576,12 @@ custom_endpoint = /log
 # Rate of events to be reported between 0 (none) and 1 (all), float
 sample_rate = 1.0
 
+# Requests per second limit, enforced an extended period, for Grafana backend log ingestion endpoint (/log)
```

Comment:
these settings should be added to https://github.com/grafana/grafana/blob/master/docs/sources/administration/configuration.md as well

Reply:
Thanks!
- [ ] describe new config options in admin docs

Reply:
done